drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/util/ModelFieldUtil.java

```diff
@@ -53,4 +54,25 @@
         }
         return availableModelFields.toArray(new ModelField[availableModelFields.size()]);
     }
+
+    /**
+     * Constructs model field with given name and data type
+     *
+     * Other three values are hardcoded as
+     *  - ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+     *  - ModelField.FIELD_ORIGIN.DECLARED,
+     *  - FieldAccessorsAndMutators.BOTH,
+     * @param fieldName name of the field
+     * @param fieldDataType data type of the field
+     * @return constructed new model field
+     */
+    public static ModelField modelField(final String fieldName,
```

Comment:
@manstis there is much more classes in `drools-wb-screens` that could use this util method, however I am not sure where I should put such util method, I don't want add dependency in each asset for `drools-wb-guided-rule-editor-client` just because of this one util method.

Any ideas where could be such method placed?

Reply:
@jomarko Heh.. well.. if you're up for the challenge then [here](https://github.com/AppFormer/uberfire/tree/master/uberfire-project-datamodel/uberfire-project-datamodel-api/src/test/java/org/appformer/project/datamodel/oracle) is probably the best place.

Reply:
@jomarko But.. it is probably well and truly out of the scope of this PR.. 

I can approve and merge this PR "as is" (now) if you like. I wasn't suggesting a thorough refactor of test code across the whole repository!

Reply:
@manstis I filed this task https://issues.jboss.org/browse/GUVNOR-3512 and assigned to me. I would like to merge this one, sure, review at first.